### ISSUE TITLE
Kogan SmarterHome 2000W Smart Glass Panel Heater appearing as "simple_switch"

### ISSUE THREAD

**Issue by domgrimm** [make-all/tuya-local#37]
As per the title, this is the device: https://www.kogan.com/au/buy/kogan-smarterhome-2000w-smart-glass-panel-heater-white/

The HA log file shows:

```
2021-07-28 17:19:20 WARNING (MainThread) [custom_components.tuya_local.config_flow] Device matches simple_switch with quality of 17%. DPS: {'1': True, '2': False, '3': 23, '4': 20, '5': 0, '7': 'High', 'updated_at': 1627456757.6805298}
2021-07-28 17:19:20 WARNING (MainThread) [custom_components.tuya_local.config_flow] Report this to https://github.com/make-all/tuya-local/issues/
```

I know '3' is the heat temperature and '4' is the current temperature, I assume '1' is auto/off, but not sure about the rest.

Any idea how to integrate this device into Tuya Local?

Thanks :)

**Comment by make-all** [comment]
It seems the same feature set as the older model that is already supported, but the DPS are rearranged.
One question - is the temperature range 16-40 like the specs on the webpage say, or 15-35 like the user manual (and in line with the older model)?

**Comment by make-all** [comment]
Thanks, if you want to test it out, there is now a config for it in the "main" branch. This needs to be installed manually from HACS by clicking on the menu button of the Tuya Local integration and selecting "Reinstall", then selecting "main" as the version from the bottom of the list.

There were a couple of reports of Kogan heaters previously working that stopped working recently, so they may have sent out this rearranged DPS layout as a firmware update to existing heaters as well.

**Comment by domgrimm** [comment]
Great thanks, I'll give it a test. Interestingly I'm able to adjust the temperature from 5-40 - the screen on the device does show "05" if I set it to 5.

**Comment by make-all** [comment]
Due to a confirmation on another issue, I've released this now as 0.9.1, so no need to go through any special steps to install the steps, just upgrade the integration to 0.9.1 and restart.

**Comment by domgrimm** [comment]
Awesome, works perfectly. Thanks for all your work!
